flocker/node/agents/functional/test_cinder.py

```diff
@@ -17,53 +17,42 @@
 
 from bitmath import Byte
 
-from ....testtools import skip_except
-from ..cinder import cinder_api, wait_for_volume
-from ..testtools import tidy_cinder_client_for_test
+from twisted.trial.unittest import SkipTest
+
 # make_iblockdeviceapi_tests should really be in flocker.node.agents.testtools,
 # but I want to keep the branch size down
 from ..test.test_blockdevice import (
-    make_iblockdeviceapi_tests, detach_destroy_volumes,
-    REALISTIC_BLOCKDEVICE_SIZE
+    make_iblockdeviceapi_tests,
+)
+from ..test.blockdevicefactory import (
+    ConfigMissing, ProviderType, get_blockdeviceapi_args,
+    get_blockdeviceapi_with_cleanup,
 )
+from ....testtools import REALISTIC_BLOCKDEVICE_SIZE
 
+from ..cinder import wait_for_volume
 
-def cinderblockdeviceapi_for_test(test_case, cluster_id):
+
+def cinderblockdeviceapi_for_test(test_case):
     """
     Create a ``CinderBlockDeviceAPI`` instance for use in tests.
 
     :param TestCase test_case: The test being run.
-    :param UUID cluster_id: The Flocker cluster ID for Cinder volumes.
     :returns: A ``CinderBlockDeviceAPI`` instance whose underlying
         ``cinderclient.v1.client.Client`` has a ``volumes`` attribute wrapped
         by ``TidyCinderVolumeManager`` to cleanup any lingering volumes that
```

Comment:
This doesn't appear to exist anymore. Also, it isn't clear why this is being removed.